susemanager-utils/susemanager-sls/src/modules/sap.py

```diff
@@ -1,10 +1,12 @@
+import glob
 import os
 import re
 
 __virtualname__ = "sap"
-__grains__ = {}
```

Comment:
This will be overridden by the loader but helps dev toos to know that it will be present and a dict. Would be nice to keep it.